Source/Godunov.H

```diff
@@ -23,6 +23,21 @@ pc_cmpflx_passive(
                         : flxrho * 0.5 * (ql + qr);
 }
 
+AMREX_GPU_DEVICE
+AMREX_FORCE_INLINE
+void
+pc_cmpflx_linpassive(
```

Comment:
this is identical with `pc_cmpflx_passive`, just call that.

Reply:
Oh good catch!